firmware/arduino/chessboard/uart.ino

```diff
@@ -84,6 +84,15 @@ bool validateMessageFromPi(volatile char * message)
             return false;
         }
     }
+    else if (message[0] == INSTRUCTION)
+    {
+        if (message[2] < ALIGN_AXIS || message[2] > RETRANSMIT)
```

Comment:
Indexing here is off; for `Instruction` messages, the last char specifies `OpType`. So that would be `message[5]`

Reply:
I don't know if there is a better way to do this, but if we change the number of `Instruction OpType`s, this line needs to be updated. So currently this is the most efficient way to do it, but it'll break later if we're not careful. Any thoughts on that?